### PR TITLE
[18.0][MIG] fieldservice_stage_server_action

### PR DESCRIPTION
**Pull request by EdgarRetes**
@max3903

### CODE REVIEW

**Comment by EdgarRetes** [comment]
@max3903 @brian10048 I need to merge this module as well for fieldservice_mobile

File: fieldservice_stage_server_action/static/description/icon.png
Comment (max3903) [inline fieldservice_stage_server_action/static/description/icon.png:1]: @EdgarRetes Can you update the icon please?

**Comment by max3903** [comment]
@EdgarRetes On the form view, I cannot move an order to a different stage and stages are not clickable...

File: fieldservice_stage_server_action/views/fsm_order.xml
Comment (max3903) [inline fieldservice_stage_server_action/views/fsm_order.xml:1]: @EdgarRetes This should be done in the `fieldservice` module

Comment (EdgarRetes) [reply fieldservice_stage_server_action/views/fsm_order.xml:1]: [https://github.com/OCA/field-service/pull/1268](url)

Comment (EdgarRetes) [reply fieldservice_stage_server_action/views/fsm_order.xml:1]: https://github.com/OCA/field-service/pull/1268

**Review by max3903 — APPROVED** [review]

**Pull request merged**